CHANGELOG.md

```diff
@@ -12,9 +12,14 @@
 ## Breaking Changes
 
 - [#722](https://github.com/oauth2-proxy/oauth2-proxy/pull/722) When a Redis session store is configured, OAuth2-Proxy will fail to start up unless connection and health checks to Redis pass
+- A bug in the Azure provider prevented it from properly passing the configured protected `--resource`
```

Comment:
Can you add in the PR link to associate with this breaking change note to tie it to the CHANGELOG entry below?

Once that's in, I'll merge 👍

Reply:
@NickMeves done